spec/factories/book.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :book, class: Book do
```

Comment:
By default factorybot looks for a class with the same name as the factory, remove `class: Book`